src/containers/ProfilePage/SectionDetailsMaybe.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { FormattedMessage } from '../../util/reactIntl';
+import { Heading } from '../../components';
+
+import css from './ProfilePage.module.css';
+
+const SectionDetailsMaybe = props => {
+  const { publicData, metadata, userFieldConfig, intl } = props;
+
+  if (!publicData || !userFieldConfig) {
+    return null;
+  }
+
+  const pickUserFields = (filteredConfigs, config) => {
+    const { key, schemaType, enumOptions, userConfig = {}, showConfig = {} } = config;
+    const { limitToUserTypeIds, userTypeIds } = userConfig;
```

Comment:
This sounds like a typo? Isn't this supposed to be _userTypeConfig_?

Reply:
Welp, good catch! Updated now.